v1/website/pages/en/index.js

```diff
@@ -51,9 +52,9 @@ class HomeSplash extends React.Component {
                   <div className="pluginRowBlock">
                     <Button
                       href={`
-                        ${siteConfig.baseUrl}docs/${
-                        this.props.language
-                      }/installation
+                        ${siteConfig.baseUrl}${getDocsUrl(
```

Comment:
Can we refactor all of those getDocsUrl(siteConfig.docsUrl) ?

Something like calling the function only once first at above.

```js
const docsUrl = getDocsUrl(siteConfig.docsUrl);
```

Then use it elsewhere